rollup.config.js

```diff
@@ -19,8 +23,41 @@ const processEnv = conf => ({
   'process.env': `(${JSON.stringify(conf)})`,
 });
 
-const vuePlugin = process.env.VUE_VERSION === 'vue3' ? vueV3 : vueV2;
-const outputDir = process.env.VUE_VERSION === 'vue3' ? 'vue3' : 'vue2';
+const vuePlugin = isVue3 ? vueV3 : vueV2;
+const outputDir = isVue3 ? 'vue3' : 'vue2';
+
+const excludeCompositionsAPI = {
+  load(id) {
+    if (id.endsWith('/src/compositions/index.js')) {
+      return '';
+    }
+    return null;
+  },
+};
+
+const applyVueCompat = {
```

Comment:
Instead of modifying vue-compat/index.js at build time, we can just replace it with index-2 or -3 file on rollup, which seems better.